default.json

```diff
@@ -4,5 +4,6 @@
   "postUpdateOptions": ["npmDedupe"],
   "rangeStrategy": "bump",
   "rebaseStalePrs": true,
-  "recreateClosed": true
+  "recreateClosed": true,
+  "schedule": ["after 10am and before 4pm every weekday"]
```

Comment:
I think we should add a comment here explaining why we are doing this. Especially because its more of an issue with the combined tooling of renovate and the merge-me action.

Reply:
@phillip-kil does this make it clearer? https://github.com/ridedott/renovate-config/pull/32/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5